spk/tt-rss/src/service-setup.sh

```diff
@@ -77,14 +71,16 @@ postinst ()
     fi
 
     # Fix permissions
-    chown ${USER} ${WEB_DIR}/${PACKAGE}/lock
-    chown ${USER} ${WEB_DIR}/${PACKAGE}/feed-icons
-    chown -R ${USER} ${WEB_DIR}/${PACKAGE}/cache
+    set_syno_permissions "${INSTALL_DIR}/var/logs" "${USER}"
+
+    set_unix_permissions "${WEB_DIR}/${PACKAGE}/lock"
```

Comment:
It seems php 5.6 does not *really* support ACLs when using the `is_writable` method and therefore causes Tiny Tiny RSS to complain about those files not being appropriately accessible.